### PR TITLE
use css3 instead of loading picture

### PR DESCRIPTION
**Pull request by songhaoreact**
delete loading.gif add use css3 for loading component

### CODE REVIEW

**Comment by dolymood** [comment]
Nice! Thanks for your PR.

File: src/components/loading/loading.vue [hunk 1]
```diff
@@ -1,6 +1,19 @@
 <template>
   <div class="cube-loading">
-    <img ref='loading' src="./loading.gif" :style="style">
+    <div class="spinner-root" :style="style">
```
Comment (dolymood) [inline src/components/loading/loading.vue]: Is span better?

File: src/components/loading/loading.vue [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+      <span class="spinner-blade"></span>
```
Comment (dolymood) [inline src/components/loading/loading.vue]: The spinner-blades can be replaced with `<i>` ?

File: src/components/loading/loading.vue [hunk 3]
```diff
@@ -27,6 +40,64 @@
...
+      width: 20px
+      height: 20px
+      .spinner-blade 
+        position: absolute
+        left: 44.5%
+        top: 37%
+        width: 10%
+        height: 25%
+        border-radius: 50%/20%
+        background-color: #8c8c8c
```
Comment (dolymood) [inline src/components/loading/loading.vue]: Use currentColor ?

File: src/components/loading/loading.vue [hunk 4]
```diff
@@ -27,6 +40,64 @@
 </script>
 <style lang="stylus" rel="stylesheet/stylus">
   .cube-loading
-    img
-      display: block
+     display: flex
```
Comment (dolymood) [inline src/components/loading/loading.vue]: display:flex should not be necessary ?

File: test/unit/specs/loading.spec.js [hunk 5]
```diff
@@ -19,7 +19,5 @@ describe('Loading.vue', () => {
     vm = instantiateComponent(Vue, Loading, {})
     expect(vm.$el.className)
       .to.equal('cube-loading')
-    expect(vm.$el.querySelector('img').src)
```
Comment (dolymood) [inline test/unit/specs/loading.spec.js]: add more tests for this change ?

**Comment by AmyFoxFN** [comment]
hi, @songhaoreact. I test this on some mobile devices, And find a problem that the loading animation direction is anticlockwise, I guess maybe caused by the negative value of animation-delay.

**Comment by AmyFoxFN** [comment]
As clockwise seems more natural, I guess maybe you can set a positive value to `animation-delay`~~O(∩_∩)O~~

File: src/components/loading/loading.vue [hunk 6]
```diff
@@ -27,6 +40,64 @@
...
+        background-color: #8c8c8c
+        animation: spinner-fade 1s linear infinite
+      .spinner-blade:nth-child(1)
+        animation-delay: -1.66667s
+        transform: rotate(30deg) translateY(-150%)
+      .spinner-blade:nth-child(2)
+        animation-delay: -1.58333s
+        transform: rotate(60deg) translateY(-150%)
+      .spinner-blade:nth-child(3)
+          animation-delay: -1.5s
```
Comment (dolymood) [inline src/components/loading/loading.vue]: 4 spaces to 2 spaces

**Comment by songhaoreact** [comment]
Thanks for the problems and suggestions you found, I need to continue optimizing

**Comment by dolymood** [comment]
Great!

**Pull request merged**